components/admin_console/audits/audits.jsx

```diff
@@ -51,18 +58,38 @@ export default class Audits extends React.PureComponent {
         );
     }
 
-    render() {
-        let content = null;
+    activityLogHeader = () => {
+        const style = {
+            display: 'inline-block',
+            margin: 0,
+        };
```

Comment:
If [style](https://github.com/mattermost/mattermost-webapp/pull/2689/files#diff-1683b148a56ec2faf3b5c5e35af1a8efL107) constant below is to stay, this can be added into it so it can be used to something like `style.header`.